File: Java/01.Java-Basic/06.ConditionalStatementsAdvancedExercise/src/Journey.java
import java.util.Scanner;

public class Journey {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        String season = scanner.nextLine();

        String destination = "";
        double price = 0.0;
        String type = "Hotel";
        if (budget <= 100) {
            destination = "Bulgaria";
            if (season.equals("summer")) {
                 price = budget * 0.3;
                 type = "Camp";
            } else if (season.equals("winter")) {
                price = budget * 0.7;
            }
        } else if (budget <= 1000) {
            destination = "Balkans";
            if (season.equals("summer")) {
                price = budget * 0.4;
                type = "Camp";
            } else if (season.equals("winter")) {
                price = budget * 0.8;
            }
        } else if (budget > 1000) {
            destination = "Europe";
            price = budget * 0.9;
        }

        System.out.printf("Somewhere in %s%n", destination);
        System.out.printf("%s - %.2f %n", type, price);
    }
}

File: JavaScript/01.JavaScriptBasic/06Nested-Conditional-Statements-Еxercise/src/Journey.js
function trip(params) {
    let budget = Number(params[0]);
    let season = params[1];

    let destination = "";
    let tripType = "";
    let tripPrice = 0.0;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season === "summer") {
            tripType = "Camp";
            tripPrice = budget * 0.3;
        } else if (season === "winter") {
            tripType = "Hotel";
            tripPrice = budget * 0.7;
        }
    } else if (budget > 100 && budget <= 1000) {
        destination = "Balkans";
        if (season === "summer") {
            tripType = "Camp";
            tripPrice = budget * 0.4;
        } else if (season === "winter") {
            tripType = "Hotel";
            tripPrice = budget * 0.8;
        }
    } else if (budget > 1000) {
        destination = "Europe";
        tripType = "Hotel";
        tripPrice = budget * 0.9;
    }

    console.log(`Somewhere in ${destination}`);
    console.log(`${tripType} - ${tripPrice.toFixed(2)}`);
}

trip(["1500", "summer"])